Makefile

```diff
@@ -78,6 +79,10 @@ ci-dependencies-start:
 		--insecure-log-all \
 		--s3compatibility.fully-compatible-listing
 
+	until [ ! -z $$(docker exec integration-sim-${BUILD_NUMBER} storj-sim network env GATEWAY_0_ACCESS) ]; do \
```

Comment:
should be `splunk-s3-tests-gateway-$$BUILD_NUMBER`

Reply:
oh, yep 🤦‍♂️ Thanks for noticing!